src/devtools/client/webconsole/components/App.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+
+// We directly require Components that we know are going to be used right away
+import ConsoleOutput from "devtools/client/webconsole/components/Output/ConsoleOutput";
+import FilterBar from "devtools/client/webconsole/components/FilterBar/FilterBar";
+import { ConsoleNag } from "ui/components/shared/Nags/Nags";
+import { FilterDrawer } from "./FilterDrawer";
+import Warning from "ui/components/shared/Warning";
+import { FC, MouseEventHandler, useEffect } from "react";
+import { useSelector } from "react-redux";
+import JSTerm from "devtools/client/webconsole/components/Input/JSTerm";
+
+/**
+ * Console root Application component.
+ */
+const App: FC = () => {
```

Comment:
:hand: Nit: the React+TS community recommends _against_ using `React.FC`:

- https://github.com/facebook/create-react-app/pull/8177
- https://fettblog.eu/typescript-react-why-i-dont-use-react-fc/

Instead the recommendation is to explicitly type `props` if it exists.

It especially doesn't add any help here, because this is a function component with no children.

You should be able to just drop the `: FC` here